easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-1.12.1-foss-2022a-CUDA-11.7.0.eb

```
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '1.12.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('RE2', '2022-06-01'),  # for torchtext
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
```

```suggestion
    ('CUDA', '11.7.0', '', SYSTEM),
    ('Python', '3.10.4'),
```
Python is missing as a dependency and is needed for '%(pyshortver)s' replacement to work in sanity check.